frigate/http.py

```diff
@@ -446,10 +449,37 @@ def latest_frame(camera_name):
         return "Camera named {} not found".format(camera_name), 404
 
 
+@bp.route("/vod/<path:path>")
+def vod(path):
```

Comment:
This builds a JSON object to be consumed by `nginx-vod-module`. Currently we are just using the existing recording path, but can expand this to add additional filters or functionality later.